homeassistant/components/asuswrt/diagnostics.py

```diff
@@ -59,16 +63,12 @@ async def async_get_config_entry_diagnostics(
             state_dict.pop("context", None)
 
         data["device"]["entities"][entity_entry.entity_id] = {
-            "name": entity_entry.name,
-            "original_name": entity_entry.original_name,
-            "disabled": entity_entry.disabled,
-            "disabled_by": entity_entry.disabled_by,
-            "entity_category": entity_entry.entity_category,
-            "device_class": entity_entry.device_class,
-            "original_device_class": entity_entry.original_device_class,
-            "icon": entity_entry.icon,
-            "original_icon": entity_entry.original_icon,
-            "unit_of_measurement": entity_entry.unit_of_measurement,
+            **async_redact_data(
```

Comment:
You can drop the `**` as `async_redact_data` is already doing it https://github.com/home-assistant/core/blob/dev/homeassistant/components/diagnostics/util.py#L33

Reply:
But I'm also adding the key `state` below.